content/stratos.md

```diff
@@ -7,33 +7,212 @@ menu:
     weight: 60
 ---
 
-# Logging in
-
-![Logging in](/img/stratos/0_login.png)
-
-![](/img/stratos/1.png)
-
-
-![](/img/stratos/2_apps.png)
-![](/img/stratos/3_new_app.png)
-![](/img/stratos/4_new_app_source.png)
-![](/img/stratos/5_deploy_github_app_1.png)
-![](/img/stratos/5_deploy_github_app_2.png)
-![](/img/stratos/5_deploy_github_app_3.png)
-![](/img/stratos/5_deploy_github_app_4.png)
-![](/img/stratos/5_deploy_github_app_5.png)
-![](/img/stratos/5_deploy_github_app_6.png)
-![](/img/stratos/6_app_aummary_and_visit.png)
-![](/img/stratos/7_browse_app.png)
-![](/img/stratos/8_app_logs.png)
-![](/img/stratos/9_edit_app_button.png)
-![](/img/stratos/10_edit_app.png)
-![](/img/stratos/11_marketplace.png)
-![](/img/stratos/12_create_service_1.png)
-![](/img/stratos/12_create_service_2.png)
-![](/img/stratos/12_create_service_3.png)
-![](/img/stratos/12_create_service_4.png)
-![](/img/stratos/13_delete_service_1.png)
-![](/img/stratos/13_delete_service_2.png)
-![](/img/stratos/14_delete_app_1.png)
-![](/img/stratos/14_delete_app_2.png)
\ No newline at end of file
+Stratos is a great way to manage your Cloud Foundry applications. While it might not be the most useful for "inner-loop" development cycles, it's a great tool for looking at configuration, logging, and resource utilization.
+
+## Logging in
+
+When you created your account, you should have been sent an email with a temporary password. 
+
+Browse to https://stratos.cap.explore.suse.dev/login and sign in with this password and the same email as you used to sign up for the Developer Community Portal (also the one that the password got sent to).
+
+![Logging In](/images/stratos/login.png)
+
+When you first log in, you will be greeted by a list of applications and pages your have "favorited". As this is your first time logging in, it will be empty.
```

Comment:
```suggestion
When you first log in, you will be greeted by a list of applications and pages you have "favorited". As this is your first time logging in, it will be empty.
```